networkit/cpp/io/test/IOGTest.cpp

```diff
@@ -1225,4 +1227,79 @@ TEST_F(IOGTest, testNetworkitBinaryZigzag) {
     }
 }
 
+TEST_F(IOGTest, testMatrixMarketReaderUnweightedUndirected) {
+    MatrixMarketReader reader;
+
+    CSRMatrix csr = reader.read("input/chesapeake.mtx");
```

Comment:
```suggestion
    MatrixMarketReader reader;

    CSRMatrix csr = MatrixMarketReader{}.read("input/chesapeake.mtx");
```
Also in other tests below.